src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -321,6 +321,37 @@ public void testForEachChild() {
             .isEqualTo(secondSibling);
     }
 
+    @Test
+    public void testForEachChildMultipleTypes() {
```

Comment:
We do not allow pure Unit tests. They fake quality and prime to produce artificial and not real main code 

Can we cover this by Input files.

If not possible I would rather use previous map().or()

Reply:
@stoyanK7 make sure that this check's test is in the target tests for pitest-utils profile in pom.xml.

Reply:
@nrmancuso I think it is there already. All `com.puppycrawl.tools.checkstyle.utils.*` classes are targeted and I see no exclusions for `TokenUtil`.

Reply:
Please re-read https://github.com/checkstyle/checkstyle/pull/12052#discussion_r971944458 carefully. Please confirm if this check's test (VariableDecl...CheckTest) is in the `targetTests` for the pitest-utils profile.

Reply:
I think I get what you mean now. We explicitly say which tests pitest should use to kill the mutation. In this case VariableDeclarationDistance uses it, so it should also be mentioned in pom.xml. Is that correct?

Reply:
To cover until method we use actual tests of Check.